db/migrate/20180501155601_update_land_registry_titles.rb

```diff
@@ -0,0 +1,11 @@
+class UpdateLandRegistryTitles < ActiveRecord::Migration[5.2]
+  def change
+    list = SubscriberList.where("title like '%Land Registry%'")
+
+    list.each do |item|
+      next if item.title.include?("HM")
```

Comment:
I'd change that to `"HM Land Registry"` to be more explicit.

Reply:
Or maybe the where to something like `title like '%Land Registry%' and title not like '%HM Land Registry%'`.

Reply:
I didn't realise I could do that - thanks @barrucadu !